### ISSUE TITLE
HTML validation fails width webp srcset 

### ISSUE THREAD

**Issue by claude27** [t3solution/t3sbootstrap#172]
HTML validation fails width webp srcset:
- with lazyload activated: "Element source is missing required attribute srcset."
- without lazyload: "Bad value for attribute srcset on element source: Expected integer but found at …lu26.jpg.webp w."

**Comment by t3solution** [comment]
Closed: problem is fixed!

**Comment by claude27** [comment]
Thank you! Unfortunately I still get the errors. Therefore I updated to the current version 5.1.7.
I now tried all 3 options in the extension configuration for lazy load ("none", "single selection in FAL" (my currenttly used configuration as in some special situations I don't want to have lazyload for an image), "for all images"). I allways get the HTML validation errors (Bad value ...). 

I have therefore deactivated my custom content templates and cleared the TYPO3 and all browser caches but it didn't help... 

For me, I solved it now by modifying these line:
if ($srcsetMode === 'w' && $processedWidth !== $candidateWidth)) {
				$widthDescriptor = $processedWidth . 'w';
			}
into 
if ($srcsetMode === 'w' && $processedWidth !== $candidateWidth **&& !is_null($processedWidth**)) {
				$widthDescriptor = $processedWidth . 'w';
			}

I didn't find out why $processedWidth is sometimes null.
